lang/es.json

```diff
@@ -326,5 +326,5 @@
   "YZECORIOLIS.ItemDeleteConfirmationTitle": "Eliminar elemento",
   "YZECORIOLIS.ItemDeleteConfirmationContent": "¿Está seguro de que desea eliminar este elemento?",
   "YZECORIOLIS.AutomaticFire": "Fuego automático",
-  "YZECORIOLIS.AutomaticFireIgnoredOnes": "Número de 1 ignorados (máx. 2)"
-}
\ No newline at end of file
+  "YZECORIOLIS.AutomaticFireModifier": "Número de 1 ignorados (máx. 2)"
```

Comment:
I think the meaning of this translation has changed. I don't think this text is relevant anymore. Google translate gives me '-2 modificador a tu tirada de ataque'